src/libraries/Microsoft.PowerFx.Core/Public/Config/SymbolTable.cs

```diff
@@ -121,7 +121,7 @@ public ISymbolSlot AddVariable(string name, FormulaType type, SymbolProperties p
 
             if (type is Types.Void)
             {
-                throw new NotSupportedException();
+                throw new InvalidOperationException($"Can't set {name} to a Void value.");

```

Comment:
>$"Can't set {name} to a Void value." [](http://example.com/codeflow?start=52&length=36)

This needs to be localized. Currently we don't have proper localization in MS.PFx.Interpreter, but we do have for MS.PFx.Core, and we should continue with it.

Reply:
My understanding could be way off here.  But I believe that the compiler type checking should catch this before we get to the symbol table, that this is only an internal error, and that makers should not see this.  All the other uses of InvalidOperationException appear to be internal and are not localized, including those from Core.  I only upgraded it from NotSupportedException as that seemed far lamer, with no real information.